frontend/components/ManageStakingPage/index.tsx

```diff
@@ -26,15 +24,8 @@ export const ManageStakingPage = () => {
       }
     >
       <WhatAreStakingContractsSection />
-      {Object.entries(
-        SERVICE_STAKING_TOKEN_MECH_USAGE_CONTRACT_ADDRESSES[Chain.GNOSIS],
-      ).map(([stakingProgram, contractAddress]) => (
-        <StakingContractSection
-          key={contractAddress}
-          stakingProgram={stakingProgram as StakingProgram}
-          contractAddress={contractAddress}
-        />
-      ))}
+      <StakingContractSection stakingProgram={StakingProgram.Beta} />
```

Comment:
yeah, this is what prompted this change for me

Idon't like the way I map the contracts array, and seems better make them separate components in case of ordering issues, or future changes to contract section components specific to a staking contract